validation/parser/face_vector_parser.py

```diff
@@ -37,4 +38,6 @@ def compute_metrics(self) -> FaceVectorMetric:
         return FaceVectorMetric(num_expected, num_missing, percentage_missing)
 
     def _build_pipeline(self) -> None:
-        self._parser_pipeline.add(self._parser_pipeline.filter_target)
+        partial_filter = partial(filter_target,
```

Comment:
The distinction between this `_build_pipeline` which doesn't use the super implementation and the RemoveParser which does is confusing and points towards there being a better abstraction. Why not have the base parser take a fully built pipeline as a constructor argument and the subclasses can take care of initializing the full pipeline object before calling `super().__init__`? In that way it's immediately clear what are the steps for each concrete parser without also having to check the superclass.